main.go

```diff
@@ -67,7 +67,7 @@ func (s *service) start() error {
 	listener, err := net.Listen("tcp", s.config.Listen)
 
 	if err != nil {
-		return err
+		panic(err)
```

Comment:
My simple change it's important for external integration tool.
Big change with logrus module need more work with refactoring.
Today akubra `return` in this case give me `exitcode 0` but when we add `panic - exitcode is 2` and it's temporary solution for me :-)